guides/micronaut-jms-aws-sqs/groovy/src/test/groovy/example/micronaut/MicronautguideSpec.groovy

```diff
@@ -0,0 +1,9 @@
+package example.micronaut
+
+
+import spock.lang.Specification
+
+
+class MicronautguideSpec extends Specification {
+    // Localstack based tests are not supported.
```

Comment:
If Spock tests are not to be implemented, simply disable them in metadata.json, eg:

https://github.com/micronaut-projects/micronaut-guides/blob/master/guides/micronaut-aws-lambda-eventbridge-event/metadata.json#L9